tests/helpers.ts

```diff
@@ -21,9 +21,10 @@ import * as nock from 'nock';
 import * as path from 'path';
 
 import * as balenaCLI from '../build/app';
-import { configureBluebird } from '../build/app-common';
+import { configureBluebird, setMaxListeners } from '../build/app-common';
 
 configureBluebird();
+setMaxListeners(25); // it appears that 'nock' adds a bunch of listeners - bug?
```

Comment:
Hmm, could be.    Perhaps related to how we are instantiating in each test ('Resetting' nock back to original state is not straighforward, and perhaps something is not cleaned up, and gets added cumulatively).  I wonder if the problem goes away if a single test is run (e.g. with `it.only(...`)